JebraAzureFunctions/JebraAzureFunctions/JebraAzureFunctions.csproj

```
@@ -3,6 +3,9 @@
     <TargetFramework>netcoreapp3.1</TargetFramework>
     <AzureFunctionsVersion>v3</AzureFunctionsVersion>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <DocumentationFile>D:\School\Ipro\Fall-2021-IPRO\JebraAzureFunctions\JebraAzureFunctions\JebraAzureFunctions.xml</DocumentationFile>
```

```suggestion
    <DocumentationFile>JebraAzureFunctions.xml</DocumentationFile>
```

This absolute file path doesn't exist on other people's machines, so we should use a relative path here